kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java

```diff
@@ -215,6 +217,22 @@ public KafkaSender build() {
     messageMaxBytes = builder.messageMaxBytes;
   }
 
+  /**
+   * Filter the properties configured for the producer by removing those not used for the Admin
+   * Client.
+   * <p>
+   * See @{@link AdminClientConfig} config properties
+   */
+  private Map<String, Object> filterPropertiesForAdminClient(Properties properties) {
```

Comment:
@ferblaca could you add a test-case to validate this method actually filters properly? that might be enough

Reply:
@jeqo sure! When I get a chance, I'll do it.